fairmq/options/FairMQProgOptions.cxx

```diff
@@ -35,6 +35,7 @@ FairMQProgOptions::~FairMQProgOptions()
 
 int FairMQProgOptions::ParseAll(const int argc, char** argv, bool allowUnregistered)
 {
+    LOG(NOLOG) << "";
```

Comment:
Alexey, obviously what you are trying to do is not a solution and not even a workaround.
Since you  based your logger on DDS's one, why don't you just implement an init method, which needs to be called in the main before using the logger?
Logger is a singleton and even multiple calls for init won't hurt.